### ISSUE TITLE
I can't add Text with Icons in Fab

### ISSUE THREAD

**Issue by RaphaelLima123** [GeekyAnts/NativeBase#2925]
I have gone through these following points

- [x] Check latest documentation: https://docs.nativebase.io/
- [x] Check for existing open/closed issues for a possible duplicate before creating a new issue: https://github.com/GeekyAnts/NativeBase/issues
- [x] Use the latest NativeBase release: https://github.com/GeekyAnts/NativeBase/releases
- [x] Check examples from NativeBase KitchenSink https://github.com/GeekyAnts/NativeBase-KitchenSink
- [x] For discussion purpose make use of NativeBase Slack: http://slack.nativebase.io/
- [x] For queries related to theme, check [Theme Variables](https://docs.nativebase.io/docs/ThemeVariables.html) from Docs and live NativeBase Theme Editor http://nativebase.io/customizer/

## Issue Description

I can't add Text next to the Fab icons. Is there any way to do this? I used styled-components/native with NativeBase components.

This is my code:

```
   <FloatingButton
        active={isActive}
        direction="up"
        containerStyle={{ flex: 5 }}
        position="bottomRight"
        onPress={() => setActive(!isActive)}
      >
        <FabIcon name="md-add" />
        <FabButtonIC>
          <IconIC name="user-plus" size={22} color="#E8DDCB" />
        </FabButtonIC>
        <FabButtonEC>
          <IconEC name="gear" size={22} color="#E8DDCB" />
        </FabButtonEC>
        <FabButtonAD>
          <IconAD name="database-refresh" size={22} color="#E8DDCB" />
        </FabButtonAD>
        <FabButtonUser>
          <IconUser name="door-open" size={22} color="#E8DDCB" />
        </FabButtonUser>
        <FabButtonExit>
          <IconExit name="logout" size={22} color="#E8DDCB" />
        </FabButtonExit>
      </FloatingButton>
```
I need the text to appear side by side with the icons on each button.

**Comment by chill-patel** [comment]
I'm also facing the same issue.

**Comment by ShavaShav** [comment]
I also need this. Is there a work around by chance?

**Comment by Elikson** [comment]
Some workaround?

**Comment by jserman** [comment]
This is the way I did it, it's a bit tacky but it looks ok.

### JSX

`<Fab
  active={isActive}
  direction="down"
  position="topLeft"
  onPress={() => setIsActive(!isActive)}
  >
    <Icon type="FontAwesome" name="car" />
    <Button style={{backgroundColor: '#DD5144'}}>
        <Icon type="FontAwesome" name="car" />
        <Text style={styles.buttonText}>Mazda</Text>
    </Button>
    <Button style={{backgroundColor: '#6daa27'}}>
        <Icon type="FontAwesome" name="car" />
        <Text style={styles.buttonText}>Peugeot</Text>
    </Button>
    <Button>
        <Icon type="AntDesign" name="plus" />
    </Button>
</Fab>`

### style

`const styles = StyleSheet.create({
	buttonText: {
		position: "absolute",
		color: "#000",
		width: 100,
		left: "120%",
		bottom: "25%"
	}
});`

Basically, I have just placed the text inside together with the icon, set its position to absolute and adjust it to my needs (to the right side of the icon). The downside is that current width is fixed to 100 which means its not responsive, but here is a starting point if you want to work on it further.

Here is what it looks like:
![fab](https://user-images.githubusercontent.com/60686452/119277896-7e5f0b00-bc22-11eb-8e36-be0c734e959a.jpeg)
